demo/layout.py

```
@@ -88,8 +88,58 @@ def freetext_tab_layout() -> html.Div:
+
+    ``snapshot_options`` is a list of ``{"label", "value"}`` dicts built by
+    the caller from a fresh ``list_snapshots()`` scan so the dropdowns never
+    go stale between tab visits.
+    """
+    _dd_style = {"marginBottom": "12px", "fontSize": "13px"}
+    return html.Div(
+        [
+            html.H2("Analys"),
+            html.Div(
+                "Jämför två snapshots: välj A och B nedan. Δ-kolumnerna "
+                "visar B − A i procentenheter (grönt = förbättring).",
```

Ruff (0.15.12)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)